modules/hilary/manifests/install/git.pp

```diff
@@ -29,7 +33,7 @@
 
         # Forcing CFLAGS for std=c99 for hiredis, until https://github.com/pietern/hiredis-node/pull/33 is resolved
         environment => ['CFLAGS="-std=c99"', 'HOME=/root'],
-        command     => 'npm install -d',
+        command     => "/usr/local/$nodejs_version/bin/npm install -d",
```

Comment:
as $nodejs_version is in a string it should have braces;
  command     => "/usr/local/${nodejs_version}/bin/npm install -d",